src/UserAdmin/src/Smart.FA.Catalog.Web/wwwroot/js/lib.js

```diff
@@ -63,11 +63,16 @@ function RegisterCultureChange() {
 function disableEnterKeyUpKeyPressOnForm() {
     var forms = document.getElementsByTagName('form');
     for (var i = 0; i < forms.length; i++) {
-        forms[i].addEventListener('keydown', ignoreEnterKeyEventHandler, true);
-        forms[i].addEventListener('keyup', ignoreEnterKeyEventHandler, true);
+        forms[i].addEventListener('keydown', disableEnterKeyEvent, true);
+        forms[i].addEventListener('keyup', disableEnterKeyEvent, true);
     }
 
     function disableEnterKeyEvent(e) {
+        var e = (e) ? e : ((event) ? event : null);
+        var node = (e.target) ? e.target : ((e.srcElement) ? e.srcElement : null);
```

Comment:
Suggested by the IDE: 
```suggestion
     var e = (e) ? e : ((Event) ? Event : null);
        var node = (e.target) ? e.target : ((e.target) ? e.target : null);
```
But more seriously, I seem to have the same behaviour with or without the method (tested on the training page only in Chrome and FIrefox in private mode). Is the enter key event blocked by default with the rich text editor fields (in quill) . Just mentionning it because I found it strange, is it the same for you @maxime-poulain ?

Reply:
Hm we loose the falback to srcElement doing so

Reply:
I see i have adjusted a bit the JS code based on this feedbck.